application/src/main/resources/thingsboard.yml

```diff
@@ -1318,8 +1317,11 @@ queue:
     max_partition_fetch_bytes: "${TB_QUEUE_KAFKA_MAX_PARTITION_FETCH_BYTES:16777216}"
     # The maximum amount of data the server will return. Records are fetched in batches by the consumer
     fetch_max_bytes: "${TB_QUEUE_KAFKA_FETCH_MAX_BYTES:134217728}"
-    request.timeout.ms: "${TB_QUEUE_KAFKA_REQUEST_TIMEOUT_MS:30000}" # (30 seconds) # refer to https://docs.confluent.io/platform/current/installation/configuration/producer-configs.html#producerconfigs_request.timeout.ms
-    session.timeout.ms: "${TB_QUEUE_KAFKA_SESSION_TIMEOUT_MS:10000}" # (10 seconds) # refer to https://docs.confluent.io/platform/current/installation/configuration/consumer-configs.html#consumerconfigs_session.timeout.ms
+    # The configuration controls the maximum amount of time the client will wait for the response of a request. By default, set 30 seconds
```

Comment:
This description is less informative. Your description does not mention that the client is in fact a Kafka Producer.  Either rephrase to simply fix the URL : https://docs.confluent.io/platform/current/installation/configuration/producer-configs.html#request-timeout-ms